.storybook/stories/components/FontStyles/Title.stories.js

```diff
@@ -0,0 +1,18 @@
+import { Heading } from 'grommet'
```

Comment:
Done, thank you!
**NOTE:** This `Title` is only used in the landing/about's banner area but nowhere else in the UI, so perhaps adding it at the page level's story would be appropriated(instead of including it as the shared building-block component).